### ISSUE TITLE
Thumbnails are not kept

### ISSUE THREAD

**Issue by gonwan** [mate-desktop/mate-desktop#203]
Steps to reproduce:
1. Clean ~/.cache/thumbnails/normal.
2. Put several images onto desktop, wait until their icons are refreshed with thumbnails.
3. Check ~/.cache/thumbnails/normal, there are generated files.
4. Wait random minutes, and press F5 to refresh your desktop, icon(thumbnail) refresh observed. If no refresh and check ~/.cache/thumbnails/normal this time, empty directory.
5. Or just logout and login, same result.
6. Not guaranteed to reproduce every time. But step 5 seems to have a higher chance.

My desktop is linuxmint 17.2 / mate 1.10.
##

**Comment by monsta** [comment]
Try this workaround:

```
gsettings set org.mate.SettingsDaemon.plugins.housekeeping active false
```

If it works for you, then you most probably have this bug: https://bugs.launchpad.net/bugs/1468794.

**Comment by flexiondotorg** [comment]
I can't reproduce this issue on Ubuntu MATE 17.04 using MATE 1.17.

**Comment by monsta** [comment]
This should be fixed in m-s-d by now (see our discussion in LP bug link).

**Comment by flexiondotorg** [comment]
I'll fix downstream ☺️

**Comment by monsta** [comment]
Yeah, I also need to send patch to Clem to fix it in Mint 17.3 :smile:
